example/vite.config.ts

```diff
@@ -1,11 +1,12 @@
 import fbteePreset from '@nkzw/babel-fbtee';
 import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
 import EnumManifest from './.enum_manifest.json' with { type: 'json' };
 import CommonStrings from './common_strings.json' with { type: 'json' };
 
 const root = process.cwd();
 
-export default {
+export default defineConfig({
```

Comment:
What does this do?

Reply:
just to help with types, see https://vite.dev/config/#config-intellisense

we could also do `export default {} satisfies UserConfig` but `defineConfig` is a bit nicer imo. It's unrelated to this PR though and slipped in by accident so I can revert it. Could add it back when we update the example